sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 3.1.0-beta.2 (Unreleased)
+
+- Renamed `Appearance` to `TextAppearance`, `Style` to `TextStyle` (previously the name of the enum for `Style.name`, and the `TextStyle` enum to `StyleName` for the sake of clarity in the type names.
+- Added `KnownStyleName`, `KnownLanguage`, `KnownSelectionMarkState`, and `KnownKeyValueType` enums to access the well-known possible values of the `StyleName`, `Language`, `SelectionMarkState`, and `KeyValueType` parameters/fields respectively.
```

Comment:
Is it a JS convention to prefix with "Known" for enums?

Reply:
Yes, as of very recently. The old type names are still there.

The type of the value is still `StyleName` (and it is just an alias to `string`), but there's a convenience layer enum `KnownStyleName` that contains entries for the known variations. It's a hacky compromise that comes from the fact that there's no way in TypeScript to say "this can be one of these strings or also any other string".